### ISSUE TITLE
Water Appears Black

### ISSUE THREAD

**Issue by Pha6se** [L8X/Roblox-Client-Optimizer#34]
Not sure why, but the water is black when enabled.

Disabled:
![RobloxScreenShot20230403_185100406](https://user-images.githubusercontent.com/101849117/229652700-c42c273d-601d-41b9-ac2c-72a7968256d5.png)

Enabled:
![RobloxScreenShot20230403_190425555](https://user-images.githubusercontent.com/101849117/229652711-99f54637-edc1-40f5-8fa3-7f9c59184283.png)

**Comment by Pha6se** [comment]
Yup! I have an RTX 3070 with the latest driver installed. I also noticed the in-game volume is quieter when RCO is enabled.

**Comment by Kaiddd** [comment]
@Pha6se if you go to studio and go to its settings, what rendering method are you using?

**Comment by Pha6se** [comment]
I leave mine on auto. 

According to this benchmark, the rendering methods only affect studio and have no effect on the client itself
![Roblox-GraphicModes](https://user-images.githubusercontent.com/101849117/229811366-8129264d-bad0-427c-863b-b5f9968aeb31.png)

**Comment by Kaiddd** [comment]
I see, good to know thank you, in that case, my only idea as to where to start, would be a lot of help from you, if l8x can't recreate the issue.

Do you know how fflags work?

**Comment by Kaiddd** [comment]
@Pha6se sorry forgot to ping.

**Comment by Pha6se** [comment]
I have a pretty good idea, I’ll try to narrow it down when I have time

**Comment by Kaiddd** [comment]
Thank you

**Comment by Pha6se** [comment]
@L8X @Kaiddd I believe I've found the FFlags causing these issues!

"FFlagTerrainVirtualTexture" < Makes the water black/white

"DFFlagVolumeSettingsUsePower2" < Makes the volume significantly lower

**Comment by Pha6se** [comment]
Yeah, it's fixed now. There's no issues with my hardware, it's the FFlag causing this. If it was my gpu, there would be problems with stock Roblox and not just with RCO
